activerecord/test/cases/associations/eager_test.rb

```diff
@@ -1340,6 +1339,10 @@ def test_deep_including_through_habtm
     # has-many :through
     david = Author.where(id: "1").eager_load(:readonly_comments).first!
     assert david.readonly_comments.first.readonly?
+
+    # belongs_to
+    post = Post.where(id: "1").eager_load(:author).first!
```

Comment:
Sorry for commenting on an old PR, but I just don’t understand this change. The association is defined on `Post` as `belongs_to :author` – no `readonly` option defined. So why does it need to be readonly after eager loading?